src/js/ripple/amount.js

```diff
@@ -176,13 +189,7 @@ Amount.prototype.ratio_human = function(denominator, opts) {
   opts = extend({ }, opts);
 
   var numerator = this;
-
-  if (typeof denominator === 'number' && parseInt(denominator, 10) === denominator) {
-    // Special handling of integer arguments
-    denominator = Amount.from_json(String(denominator) + '.0');
-  } else {
-    denominator = Amount.from_json(denominator);
-  }
+  denominator = Amount.from_json(noIntegers(denominator));
```

Comment:
Yeah, this should be fixed. I think it deserves it's own ticket and pull request because it will probably require a lot of changes to the tests: https://ripplelabs.atlassian.net/browse/RLJS-215

We need to be really careful with this though, I'm not familiar with all the things this change could break.